Solutions/ForescoutHostPropertyMonitor/Analytic Rules/ForeScout-DNSSniffEventMonitor.yaml

```diff
@@ -0,0 +1,30 @@
+id: d272e277-f285-4dbc-ae2d-7f65ba64a79e
+name: Forescout-DNS_Sniff_Event_Monitor
+description: 'This rule creates an incident when more than certain number of Dnsniff events are generated from a host'
+severity: Medium
+requiredDataConnectors: []
```

Comment:
@rushriva and @NikTripathi  - Shouldn't this have a required data connector as part of a Solution?

Reply:
This solution makes use of Sentinel REST API to upload and download data to Log Analytics. Because of this reason, a Data Connector is not needed for solution.